lib/inertia_rails/controller.rb

```diff
@@ -20,6 +20,26 @@ def inertia_share(**args, &block)
           InertiaRails.share_block(block) if block
         end
       end
+
+      def inertia_instance_props
```

Comment:
Is this method leftover from an earlier iteration? I don't see it called anywhere else in the PR

Reply:
thank you, great catch!